src/main/java/com/foodybuddy/dao/OrderDAO.java

```diff
@@ -12,10 +13,18 @@
 	/**
 	 * Insert.
 	 *
-	 * @param c the c
-	 * @return 
+	 * @param order the order
+	 * @return the order
+	 */
+	public Order insert(Order order);
+	
+	/**
+	 * Save. Returns the Identifier immediately
+	 *
+	 * @param order the order
+	 * @return the order
 	 */
-	public Order insert(Order c);
+	public Order save(Order order);
```

Comment:
This saves the order immediately and give the orderId . We don't need to wait till the transaction completed. This particular scenario is used in placing order